File: TouchPOS_API/Models/TenderModels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
    public class TenderModels
    {
        public class Sel_Tender
        {
            public int TID { get; set; }
            public string CODE { get; set; }
            public string NAME { get; set; }
            public string VENDERCODE { get; set; }
            public string VENDERNAME { get; set; }
            public string TYPE { get; set; }
            public string CERRENCY { get; set; }
            public string PAYMENT_METHOD { get; set; }
            public decimal VALUE_CONVERT { get; set; }
            public string Server { get; set; }
            public int SEQ { get; set; }
            public string TERMINALID { get; set; }
            public string POSID { get; set; }
            public string MERCHAND_ID { get; set; }
            public string APP_ID { get; set; }
            public string CLIENT_KEY { get; set; }
            public string CLIENT_SECRET { get; set; }

            public string PREFIX_BARCODE { get; set; }
            public string REMARK1 { get; set; }
            public string REMARK2 { get; set; }
            public string REMARK3 { get; set; }
            public string SCAN_TYPE { get; set; }
            public string VENDER_SHOP_ID { get; set; }

            public Sel_Tender()
            {
                TID = 0;
                CODE = "";
                NAME = "";
                VENDERCODE = "";
                VENDERNAME = "";
                TYPE = "";
                CERRENCY = "";
                PAYMENT_METHOD = "";
                VALUE_CONVERT = 0;
                Server = "";
                SEQ = 0;
                TERMINALID = "";
                POSID = "";
                MERCHAND_ID = "";
                APP_ID = "";
                CLIENT_KEY = "";
                CLIENT_SECRET = "";
                PREFIX_BARCODE = "";
                REMARK1 = "";
                REMARK2 = "";
                REMARK3 = "";
                SCAN_TYPE = "";
                VENDER_SHOP_ID = "";


            }
        }
    }
}

File: TouchPOS/Models/TenderModels.vb
Imports MD
Public Class TenderModels
    Public Class value
        Public Respon As New ResponseModels
        Public Data As List(Of Sel_Tender)

    End Class
    Public Class Sel_Tender

        Private _TID As Integer
        Public Property TID() As Integer
            Get
                Return _TID
            End Get
            Set(value As Integer)
                _TID = value
            End Set
        End Property
        Private _CODE As String
        Public Property CODE() As String
            Get
                Return _CODE
            End Get
            Set(value As String)
                _CODE = value
            End Set
        End Property

        Private _NAME As String
        Public Property NAME() As String
            Get
                Return _NAME
            End Get
            Set(value As String)
                _NAME = value
            End Set
        End Property
        Private _VENDERCODE As String
        Public Property VENDERCODE() As String
            Get
                Return _VENDERCODE
            End Get
            Set(value As String)
                _VENDERCODE = value
            End Set
        End Property
        Private _VENDERNAME As String
        Public Property VENDERNAME() As String
            Get
                Return _VENDERNAME
            End Get
            Set(value As String)
                _VENDERNAME = value
            End Set
        End Property
        Private _TYPE As String
        Public Property TYPE() As String
            Get
                Return _TYPE
            End Get
            Set(value As String)
                _TYPE = value
            End Set
        End Property

        Private _CERRENCY As String
        Public Property CERRENCY() As String
            Get
                Return _CERRENCY
            End Get
            Set(value As String)
                _CERRENCY = value
            End Set
        End Property

        Private _PAYMENT_METHOD As String
        Public Property PAYMENT_METHOD() As String
            Get
                Return _PAYMENT_METHOD
            End Get
            Set(value As String)
                _PAYMENT_METHOD = value
            End Set
        End Property


        Private _VALUE_CONVERT As Decimal
        Public Property VALUE_CONVERT() As Decimal
            Get
                Return _VALUE_CONVERT
            End Get
            Set(value As Decimal)
                _VALUE_CONVERT = value
            End Set
        End Property
        Private _Server As String
        Public Property Server() As String
            Get
                Return _Server
            End Get
            Set(value As String)
                _Server = value
            End Set
        End Property

        Private _SEQ As Integer
        Public Property SEQ() As Integer
            Get
                Return _SEQ
            End Get
            Set(value As Integer)
                _SEQ = value
            End Set
        End Property

        Private _TERMINALID As String
        Public Property TERMINALID() As String
            Get
                Return _TERMINALID
            End Get
            Set(value As String)
                _TERMINALID = value
            End Set
        End Property

        Private _POSID As String
        Public Property POSID() As String
            Get
                Return _POSID
            End Get
            Set(value As String)
                _POSID = value
            End Set
        End Property

        Private _MERCHAND_ID As String
        Public Property MERCHAND_ID() As String
            Get
                Return _MERCHAND_ID
            End Get
            Set(value As String)
                _MERCHAND_ID = value
            End Set
        End Property


        Private _APP_ID As String
        Public Property APP_ID() As String
            Get
                Return _APP_ID
            End Get
            Set(value As String)
                _APP_ID = value
            End Set
        End Property

        Private _CLIENT_KEY As String
        Public Property CLIENT_KEY() As String
            Get
                Return _CLIENT_KEY
            End Get
            Set(value As String)
                _CLIENT_KEY = value
            End Set
        End Property
        Private _CLIENT_SECRET As String
        Public Property CLIENT_SECRET() As String
            Get
                Return _CLIENT_SECRET
            End Get
            Set(value As String)
                _CLIENT_SECRET = value
            End Set
        End Property

        Private _PREFIX_BARCODE As String
        Public Property PREFIX_BARCODE() As String
            Get
                Return _PREFIX_BARCODE
            End Get
            Set(value As String)
                _PREFIX_BARCODE = value
            End Set
        End Property
        Private _REMARK1 As String
        Public Property REMARK1() As String
            Get
                Return _REMARK1
            End Get
            Set(value As String)
                _REMARK1 = value
            End Set
        End Property
        Private _REMARK2 As String
        Public Property REMARK2() As String
            Get
                Return _REMARK2
            End Get
            Set(value As String)
                _REMARK2 = value
            End Set
        End Property
        Private _REMARK3 As String
        Public Property REMARK3() As String
            Get
                Return _REMARK3
            End Get
            Set(value As String)
                _REMARK3 = value
            End Set
        End Property
        Private _SCAN_TYPE As String
        Public Property SCAN_TYPE() As String
            Get
                Return _SCAN_TYPE
            End Get
            Set(value As String)
                _SCAN_TYPE = value
            End Set
        End Property

        Private _VENDER_SHOP_ID As String
        Public Property VENDER_SHOP_ID() As String
            Get
                Return _VENDER_SHOP_ID
            End Get
            Set(value As String)
                _VENDER_SHOP_ID = value
            End Set
        End Property

    End Class
End Class
